Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/Compatible/WebBaseCompatibilityAdapter.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Windows;
+using System.Windows.Data;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF.Compatible
```

Comment:
This namespace doesn't make any sense to me. Shouldn't this also go into the WebView or Controls namespace per https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/framework-design-guidelines-digest.md?

Reply:
I looked at this from a consumer standpoint.  I can't tell you how many times I've seen framework classes with really similar names that I've wanted to alias just to remember why I chose one over the other.  I'd rather err on the side of over-qualification in the name and namespace.  Not married to it, so I'll change it... but... that's where it came from.